.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
 .env
 node_modules
-build
\ No newline at end of file
+build
+jest.config.js 
```

Comment:
jest.config.js shouldn't be in .gitignore because without that file when I was running npm test, tests was failing

Reply:
I removed it from the .gitgnore file.